reana_job_controller/kubernetes_job_manager.py

```diff
@@ -66,17 +66,25 @@ def __init__(self, docker_img=None, cmd=None, env_vars=None, job_id=None,
         :type job_name: str
         :param kerberos: Decides if kerberos should be provided for job.
         :type kerberos: bool
+        :param kubernetes_uid: User ID for job container.
+        :type kubernetes_uid: int
         """
         super(KubernetesJobManager, self).__init__(
-                         docker_img=docker_img, cmd=cmd,
-                         env_vars=env_vars, job_id=job_id,
-                         workflow_uuid=workflow_uuid,
-                         workflow_workspace=workflow_workspace,
-                         job_name=job_name)
+            docker_img=docker_img,
+            cmd=cmd,
+            prettified_cmd=prettified_cmd,
+            env_vars=env_vars,
+            workflow_uuid=workflow_uuid,
+            workflow_workspace=workflow_workspace,
+            job_name=job_name)
         self.compute_backend = "Kubernetes"
         self.cvmfs_mounts = cvmfs_mounts
         self.shared_file_system = shared_file_system
         self.kerberos = kerberos
+        if kubernetes_uid:
```

Comment:
What if user specifies `kubernetes_uid=0` to become root and do malicious things? Perhaps we should implement the check I had mentioned in https://github.com/reanahub/reana-workflow-controller/issues/269 -- i.e. check if the values are below or above 100 and accept only those -- here in R-J-C...